### PR TITLE
383 notebook reviews

### LINKED ISSUE #383: Notebook Review #5
1. cPW_kappa_calculation_demo.ipynb https://github.com/Qiskit/qiskit-metal/blob/main/tutorials/6%20Analysis/CPW_kappa_calculation_demo.ipynb

2. Capacitance and Frequency Control.ipynb https://github.com/Qiskit/qiskit-metal/blob/main/tutorials/6%20Analysis/Capacitance%20and%20Frequency%20Control.ipynb

3. Scan Options - https://github.com/Qiskit/qiskit-metal/blob/main/tutorials/6%20Analysis/S%20Param%20Simulation.ipynb

4. 6 - Analyzing a transmon  https://github.com/Qiskit/qiskit-metal/blob/main/tutorials/6%20Analysis/Analyzing%20a%20transmon.ipynb

### PR DESCRIPTION
**Pull request by ThomasGM4**
### What are the issues this pull addresses (issue numbers / links)?

Closes #383 

### Did you add tests to cover your changes (yes/no)?

No

### Did you update the documentation accordingly (yes/no)?

Yes

### Summary

Notebook reviews and modifications
Added bib file
Made minor modifications to some different methods.

### Details and comments

Notebooks should be fully functional, small errors have been fixed.

Added a .bib file for citing of Metal. Link in FAQ pointing to said file to be later added (such as in https://qiskit.org/documentation/faq.html)

Some small changes to a few methods, mainly modifying default values to more design appropriate options. One changed the number of expected inputs, which may need an update to tests.

### CODE REVIEW

**Review by JeremyDrysdale-ibm — DISMISSED** [review]
Not worth holding the PR up for, but I have one question about the `.bib` year noted below.

File: tutorials/6 Analysis/S Param Simulation.ipynb [hunk 1]
```diff
@@ -311,8 +318,8 @@
     "First parameter: List of components to render (empty list if rendering whole Metal design) <br>\n",
```
Comment (priti-ashvin-shah-ibm) [inline tutorials/6 Analysis/S Param Simulation.ipynb:318]: Above, regarding cell for open_ansys(), suggest alternate text.

Skip the next cell if Ansys is already open and with a project loaded.

in bold:
**Wait for Ansys to open up completely before proceeding to the next cell.**

File: Qiskit_Metal.bib [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@misc{Qiskit_Metal,
+author = {Zlatko K. Minev and Thomas G. McConkey and Jeremy Drysdal and Priti Shah and Dennis Wang and Marco Facchini and Grace Harper and John Blair and Helena Zhang and Nick Lanzillo and Sagarika Mukesh and Will Shanks and Chris Warren},
+title = {Qiskit Metal: An Open-source Framework for Quantum System Design},
+year = {2020},
```
[the remark this answers is not in the document]
Comment (ThomasGM4) [reply Qiskit_Metal.bib:4]: That is a good question. Not sure what the rule is for open source software, when first mentioned publicly? When it becomes open source? When it hits version 1.0?

File: tutorials/6 Analysis/Analyzing a transmon.ipynb [hunk 3]
```diff
@@ -53,7 +56,6 @@
   },
   {
    "cell_type": "markdown",
-   "id": "southeast-stock",
```
[the remark this answers is not in the document]
Comment (ThomasGM4) [reply tutorials/6 Analysis/Analyzing a transmon.ipynb:56]: Didn't quite follow what you meant for this.

[the remark this answers is not in the document]
Comment (priti-ashvin-shah-ibm) [reply tutorials/6 Analysis/Analyzing a transmon.ipynb:56]: "Note that the design size extends from the origin into the first quadrant."

The above text is written below.

**Review by JeremyDrysdale-ibm — DISMISSED** [review]
LGTM

**Comment by grace-harper-ibm** [comment]
Is `https://github.com/Qiskit/qiskit-metal/blob/main/Qiskit_Metal.bib` supposed to be an accessible website from the browser? I was unable to access it

**Review by priti-ashvin-shah-ibm — APPROVED** [review]
LGTM

**Review by JeremyDrysdale-ibm — DISMISSED** [review]
LGTM

**Review by grace-harper-ibm — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -209,7 +209,7 @@ Then open the applicable settings.json in your VS Code. (See how to open command
 
 ## Authors and Citation
 
-Qiskit Metal is the work of [many people](https://github.com/Qiskit/qiskit-metal/pulse/monthly) who contribute to the project at different levels. Metal was conceived and developed by [Zlatko Minev](zlatko-minev.com) at IBM; then co-led with Thomas McConkey. If you use Qiskit, please cite as per the included [BibTeX file](https://github.com/Qiskit/qiskit/blob/master/Qiskit.bib). For icon attributions, see [here](/qiskit_metal/_gui/_imgs/icon_attributions.txt).
+Qiskit Metal is the work of [many people](https://github.com/Qiskit/qiskit-metal/pulse/monthly) who contribute to the project at different levels. Metal was conceived and developed by [Zlatko Minev](zlatko-minev.com) at IBM; then co-led with Thomas McConkey. If you use Qiskit Metal, please cite as per the included [BibTeX file](https://github.com/Qiskit/qiskit-metal/blob/main/Qiskit_Metal.bib). For icon attributions, see [here](/qiskit_metal/_gui/_imgs/icon_attributions.txt).
```
[the remark this answers is not in the document]
Comment (priti-ashvin-shah-ibm) [reply README.md:212]: I checked also, I was not able to access https://github.com/Qiskit/qiskit-metal/blob/main/Qiskit_Metal.bib.

File: tutorials/6 Analysis/Analyzing a transmon.ipynb [hunk 5]
```diff
@@ -2,16 +2,21 @@
  "cells": [
```
[the remark this answers is not in the document]
Comment (ThomasGM4) [reply tutorials/6 Analysis/Analyzing a transmon.ipynb:2]: Hmmm, I am now also getting that error.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ThomasGM4) [reply README.md:212]: Because it is pointing at main, which this branch has not yet merged into.

**Review by priti-ashvin-shah-ibm — APPROVED** [review]
LGTM

**Review by grace-harper-ibm — APPROVED** [review]
LGTM

**Review by JeremyDrysdale-ibm — APPROVED** [review]
LGTM

**Pull request merged**